src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/intro";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");

```

Comment:
Storing sensitive information such as the database username and password in the code is a bad practice due to security reasons. It's better to store such information in an external configuration file or environment variables.